app/Models/Campaign.php

```diff
@@ -210,6 +210,15 @@ public function enabled(string $module): bool
         return (bool) CampaignCache::settings()->get($module);
     }
 
+    /**
+     * Determine if a campaign has a module enabled or not
+     *
+     */
+    public function modules()
+    {
+        return CampaignCache::settings();
```

Comment:
This smell of being a terrible idea (the cache isn't always  up to date in the queue since data is cache differently in the queue to be faster). Also adding a function on the campaign model means it will load a few more bytes on every single request, which results in several megabytes of extra ram needed for nothing